core/src/main/java/org/openedx/core/ui/theme/AppColors.kt

```diff
@@ -19,9 +21,16 @@ data class AppColors(
     val textFieldText: Color,
     val textFieldHint: Color,
 
-    val buttonBackground: Color,
-    val buttonSecondaryBackground: Color,
-    val buttonText: Color,
+    val primaryButtonBackground: Color,
+    val primaryButtonText: Color,
+    val primaryButtonBorder: Color,
+    val primaryButtonBorderedText: Color,
+
+    val secondaryButtonBackground: Color,
```

Comment:
What about adding some comments for clarity?

```suggestion
    val primaryButtonBorderedText: Color,

    //Default secondary button is the same as the primary button.
    //But you can change it if your brand has 2 accent colors
    val secondaryButtonBackground: Color,
```

Reply:
@volodymyr-chekyrta 
What are your thoughts on this?
```
// The default secondary button styling is identical to the primary button styling.
// However, you can customize it if your brand utilizes two accent colors.
```

Reply:
addressed.

Reply:
Sounds great to me! 👍